### DISCUSSION TITLE
期末報告與期末考

### DISCUSSION THREAD

**Discussion by Aaron0810** [1101-GS4719-C/Python-Community-GS4719-C#1499] [閒聊八卦]
大家早安，我來關心一下大家的期末狀況好了，各位的期末都是在上周(第16周)就結束還是要到真正學期最後一周才結束，我自己是還有三科考試要在最後一周才考，所以還要再撐兩個禮拜才能放假，再加上有些課還要做期末報告，其實事情蠻多的，也順便問一下這裡的大家是期末考比較多還是期末告報比較多呢?

**Comment by 111607004** [comment]
辛苦了，要撐到最後一周，跟上學期的我一樣QQ我記得我上學期的最後一科是18周的星期五考，到最後學校都沒人了，只剩下自己在那邊奮鬥，真的好折磨！！

**Comment by Elaine2899** [comment]
我這學期的課程只有兩堂課需要報告，一堂以為電影方式呈現，另一堂則是python的期末報告，加上期末考最後一科是在17周的第一天、為電影在17周第三天報告，因此python的報告可以有較為充裕的時間準備，希望下學期的課程也能這麼愜意。

**Comment by PaulSu0905** [comment]
我這學期是期末考較多，而且剛好所有考試都集中在同一周，一到五都有考試，陷入無止盡的考試地獄，讓我好懷念可以睡到自然醒的時光，但也只能怪我自己平常太混，期末才需要讀的那麼辛苦。

**Comment by 111607004** [comment]
這學期我期末考比較多，而且大多都集中在16、17周，所以最近比較辛苦，但最後一科是在下禮拜一，所以下禮拜可以提早放假(好欸!)。而唯一的期末報告就是python課的期末報告。

**Comment by Brownnnn78** [comment]
這學期一開始很多老師都預定在16週考試，然而進度中間放假比較多，最後有的科目16週考有的延到17週，原本還有到18週但老師聽我們意見17週考，但18週依然要上課，那些16週考試的都出了好多期末報告，所以其實也沒有多快樂⋯⋯反而從16週開始就要一直枯燥乏味的備考日子。

**Comment by Evening218** [comment]
改成16+2的好處就是不會有爆炸的期末考周,但是我覺得壞處就是當有的課在16周結束,有的要拖到18周,就會變成一個很漫長的痛苦過程,感覺每一周都是期末考周QAQ

**Comment by MaxChi870** [comment]
多半都是期末考，彈性16周的好處是課程在舉行期末考時被拆分成不同周的機會變大了，但相對的大部分的課程內容都是原先計畫18周的教學，導致內容的吸收變得相較急促，還好一切都過了，預祝各位暑假假期愉快。

**Comment by Miinteaa** [comment]
我這學期是期末考較多，但其實比起選擇題、手寫題考試，我更喜歡做上機的考試或者說project ，只是這樣就會變成18週deadline 截止前都無法放假，真的好兩難🥲

**Comment by zhiron544** [comment]
我覺得這篇文章很有共鳴，我也是還有兩科要考和一個報告要交，感覺好累啊。我覺得期末考試應該減少一些，讓學生有更多時間學習和休息。我希望你能順利通過所有的考試和報告，加油！

**Comment by Alex11392** [comment]
期末考比較多，但是在一周內就全部結束了，速戰速決，十分愉快

**Comment by 110302001** [comment]
我這學期期末報告和考試各佔一半，但老實說我比較喜歡報告放在期中，期末考試，我覺得期末考最好是分數馬上出來那種，不然暑假還要擔心報告和考試分數，心好累。

**Comment by XiangLuen** [comment]
這學期的課程非常充實，有一半的課需要期末報告，然後有些甚至會有期末考加期末報告，而且這些課不修還無法畢業，我是覺得期中考再加期末報告是最好的，畢竟報告可以讓AI整理，然後剩下時間可以讀其他科目，或是檢查內容正確性

**Comment by hollowha** [comment]
我也覺得這學期的課很讚，主要是要留夠多的空白時間給自己，然後多選一些自己有興趣的😁
